### DISCUSSION TITLE
video+mp3

### DISCUSSION THREAD

**Discussion by knxwille** [jely2002/youtube-dl-gui#250] [General]
Hello, 
First of all, thank you for your nice app.
I have old tv. I can watch mp4 videos but if sound track is mp3. My tv doesn't support aac/ac3 audio tracks.
Is there any way to download audio track as mp3? 
I can re-encode video (copy video,convert audio to mp3 then mux them to mp4 container) with ffmpeg but I want to do it auto with the app. 
Is there any way to do that?

Thanks again.
Knx.

**Comment by olegshulyakov** [comment]
@knxwille There are options for output:
<img width="477" alt="image" src="https://user-images.githubusercontent.com/4524455/174337395-aea7e860-c3ab-4f65-91c9-915d9984aa25.png">

**Comment by jely2002** [comment]
In the newer versions there's still an output setting. By default it will re-encode audio to mp3 so files are compatible with most players.
Its also possible to force a specific codec if that works better with a tv for example.
